ksqldb-common/src/test/java/io/confluent/ksql/schema/ksql/SchemaConvertersTest.java

```diff
@@ -162,8 +166,7 @@ public void shouldHaveConnectTestsForAllSqlTypes() {
         .map(SqlType::baseType)
         .collect(Collectors.toSet());
 
-    final ImmutableSet<SqlBaseType> allTypes = ImmutableSet.copyOf(SqlBaseType.values());
-
+    final ImmutableSet<SqlBaseType> allTypes = ImmutableSet.copyOf(ArrayUtils.removeElement(SqlBaseType.values(), SqlBaseType.LAMBDA));
```

Comment:
Ditto as above - I think if we implement this properly I don't see why lambdas should be any different than array types at this point. (I'll stop commenting this, but there are various places where we omit lambdas where I don't think we should treat them differently in our type system)